src/routers/v2/DelaysRouter.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import ApplicationRouter from '../../appdev/ApplicationRouter';
+import RealtimeFeedUtils from '../../utils/RealtimeFeedUtils';
+
+class DelaysRouter extends ApplicationRouter<Object> {
+  constructor() {
+    super(['POST']);
+  }
+
+  getPath(): string {
+    return '/delays/';
+  }
+
+  async content(req): Promise<any> {
+    const delays = req.body.data;
+    const rtf = await RealtimeFeedUtils.fetchRTF();
+    const delaysPromises = delays.map(async (stopAndTripID) => {
+      const tripID = await stopAndTripID.tripID;
+      const stopID = await stopAndTripID.stopID;
+      const res = await RealtimeFeedUtils.getDelayInformation(stopID, tripID, rtf);
+      if (res) {
+        stopAndTripID.delay = res.delay;
+      } else {
+        stopAndTripID.delay = null;
+      }
```

Comment:
nit: `stopAndTripID.delay = res ? res.delay : null`

Reply:
debatable whether or not you accept this :joy:, I think the way you have it is more readable!